CMakeLists.txt

```diff
@@ -48,9 +48,37 @@ endif()
 if (NOT BLT_CXX_STD)
     set(BLT_CXX_STD "c++17" CACHE STRING "")
 endif()
-set(ENABLE_ASTYLE      OFF CACHE BOOL "")
-set(ENABLE_CLANGQUERY  OFF CACHE BOOL "")
-set(ENABLE_UNCRUSTIFY  OFF CACHE BOOL "")
+
+# These BLT tools are not used in Serac, turn them off
+set(_unused_blt_tools
+    CLANGQUERY
+    VALGRIND
+    ASTYLE
+    CMAKEFORMAT
+    UNCRUSTIFY
+    YAPF)
+foreach(_tool ${_unused_blt_tools})
+    set(ENABLE_${_tool} OFF CACHE BOOL "")
+endforeach()
+
+# These BLT tools are only used by Serac developers, so turn them off
+# unless an explicit executable path is given
+set(_used_blt_tools
+    CLANGFORMAT
+    CLANGTIDY
+    CPPCHECK
+    DOXYGEN
+    SPHINX)
+foreach(_tool ${_used_blt_tools})
+    if(NOT ${_tool}_EXECUTABLE)
+        set(ENABLE_${_tool} OFF CACHE BOOL "")
+    else()
+        set(ENABLE_${_tool} ON CACHE BOOL "")
```

Comment:
My understanding is that these are ON by default, so is this just for clarity/explicitness?

Reply:
Explicitness and I'm thinking of turning them off by default due to the large set we support in BLT at this point.  Easier to turn on the ones you want then turn off all the ones you don't.